manifests/m/MicrosoftCorporation/VisualFoxProODBCDriver/1.0.0/MicrosoftCorporation.VisualFoxProODBCDriver.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.VisualFoxProODBCDriver
+PackageVersion: 1.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
+  InstallerUrl: http://www.topol.eu/files/download/other/VFPODBC.msi

```

```suggestion
  InstallerUrl: http://www.topol.eu/files/download/other/VFPODBC.msi
```
Only official URLs are allowed.

The VFPODBC driver is no longer supported. We strongly recommend using the Visual FoxPro OLE DB provider as a replacement. Please refer to the following article for more information and related links to issues when using the VFPODBC driver: https://support.microsoft.com/kb/277772.

Looks like download is no longer available at https://docs.microsoft.com/en-us/previous-versions/visualstudio/foxpro/mt490121(v=msdn.10) but "Visual FoxPro OLE DB" is